src/sqlancer/StatementExecutor.java

```diff
@@ -70,7 +70,7 @@ public void executeStatements() throws Exception {
                     success = globalState.executeStatement(query);
                 } while (nextAction.canBeRetried() && !success
                         && nrTries++ < globalState.getOptions().getNrStatementRetryCount());
-            } catch (IgnoreMeException e) {
+            } catch (IgnoreMeException ignored) {
```

Comment:
Generally, it is not recommended to use an empty catch to silence an exception.

But in our case, do we need to print the error message or just ignore it?